RandomMonomialIdeals.m2

```diff
@@ -199,7 +199,7 @@ TEST ///
     -- Check every monomial is generated
     L=(randomGeneratingSets(2,3,1.0,1))#0
     R=ring(L#0)
-    assert(L=={R_0,R_1,R_0^2,R_0*R_1,R_1^2,R_0^3,R_0^2*R_1,R_0*R_1^2,R_1^3})
+    assert(sort L==sort {R_0,R_1,R_0^2,R_0*R_1,R_1^2,R_0^3,R_0^2*R_1,R_0*R_1^2,R_1^3})
```

Comment:
By the way, Sets are compared using the === instead of == .  (!)
See here:
http://www.math.uiuc.edu/Macaulay2/doc/Macaulay2-1.9/share/doc/Macaulay2/Macaulay2Doc/html/___Set.html